### PR TITLE
build(paperclip): support multiple platforms

### PR DESCRIPTION
**Pull request by tiagolobocastro**
test(bdd/thin): fix racy thin prov test
    
    Add retry waiting for condition to be met.
    
    Signed-off-by: Tiago Castro <tiagolobocastro@gmail.com>

---

    build(paperclip): support multiple platforms
    
    Supports building paperclip from multiple plaforms.
    Also updates old openapi update script to update paperclip.
    
    Signed-off-by: Tiago Castro <tiagolobocastro@gmail.com>

### CODE REVIEW

**Review by abhilashshetty04 — APPROVED** [review]

File: shell.nix [hunk 1]
```diff
@@ -33,6 +33,7 @@ mkShell {
     jq
     llvmPackages.libclang
     nixpkgs-fmt
+    nix
     openapi-generator-cli
```
Comment (abhilashshetty04) [inline shell.nix:37]: openapi-generator-cli still needed?

Comment (tiagolobocastro) [reply shell.nix:37]: Yeah but not our fork, just the upstream version for the python code

**Comment by tiagolobocastro** [comment]
bors merge

**Comment by tiagolobocastro** [comment]
bors merge

**Comment by tiagolobocastro** [comment]
bors cancel

**Comment by tiagolobocastro** [comment]
bors merge

**Pull request merged**